### ISSUE TITLE
Empty map, isn't an item that has been accounted for in Hut's. 

### ISSUE THREAD

**Issue by crysallis** [ldtteam/minecolonies#8558]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest MineColonies Verison?

- [X] I am running the latest **alpha** version of MineColonies for my Minecraft version.

### Did you check on the Wiki? or ask on Discord?

- [X] I checked the MineColonies Wiki and made sure my issue is not covered there. Or I was sent from discord to open an issue here.

### Minecraft Version

1.18

### MineColonies Version

minecolonies-1.18.2-1.0.919-ALPHA

### Structurize Version

structurize-1.18.2-1.0.424-ALPHA

### Related Mods and their Versions

[mods.txt](https://github.com/ldtteam/minecolonies/files/9313676/mods.txt)

### Current Behavior

The level 4 Dark Oak University calls for an Empty Map,

### Expected Behavior

Items used in huts should be generally buildable by the colonists

### Reproduction Steps

Attempt to teach someone to make an empty map, doesn't allow you to do so.  JEI also doesn't list anything for that item

### Logs

no logs really needed for this

### Anything else?

I know it's pretty rare, I did ask in the discord before hand.  I made this myself but figured someone else may run into it so I've submitted it.

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by uecasm** [comment]
If you look at the logs you didn't include, you'll see why.  There's something weird about the recipe for empty maps that makes them incompatible, and it logs that.  It might eventually be fixable, but for now this is intended, since there's not really any great need to be autocrafting maps anyway.

**Comment by crysallis** [comment]
I asked others to validate the issue before I submitted so with is being so reproduceable, my logs wouldn't have shed much light on the issue.  But I'll include them next time.  It also would still have been a bug... Whether or not I saw why... So understanding the why here isn't really for me, that's more for those that dev the product.  If this is working as intended, yet broken then fine.  It is what it is ... Just seems like if the hut blocks are gonna display/use an item... The colony should be able to create it.  I was just trying to let someone know is all.

**Comment by uecasm** [comment]
That's fine.  But it's not mandatory for the colony to produce things that are used in builds -- there's no way to produce coral, for example, but it's included in some high level buildings.  Some exploration and player-gathering is expected, especially at the higher levels.

**Comment by crysallis** [comment]
You're right they don't have to make them all... never said they did... only said I was saying something because it was suggested in the discord that I do it... I don't really need to banter back and forth about it... I'm not trying to argue for or against it... only that it seems like there should be something for items that are used in the mod.  I'm aware of player gathering etc... if you are the person that would make this change and you don't want to... fine just say that.. it's not a contest, it's just one users observation.  And case in point... there is in fact is a recipe for coral... using coral essence from mystical agriculture...
